packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -124,6 +129,11 @@ export class BlobManager {
      */
     private readonly pendingBlobs: Map<string, PendingBlob> = new Map();
 
+    /** If true, throw an error when a tombstone data store is retrieved. */
+    private readonly throwOnTombstoneUsage: boolean;
+    /** Tombstone is a temporary feature that imitates a blob getting swept by garbage collection. */
```

Comment:
Maybe start this comment by saying this stores ids of tombstoned blobs and then what tombstone is.